src/v/cluster/members_backend.cc

```diff
@@ -221,6 +221,18 @@ ss::future<> members_backend::calculate_reallocations(update_meta& meta) {
         co_return;
     case members_manager::node_update_type::recommissioned:
         co_await calculate_reallocations_after_recommissioned(meta);
+        if (
```

Comment:
I'm not sure about this behavior.

If someone is recommissioning (i.e. cancelling a decommission), it may not be because they really want to keep the node permanently: maybe something went wrong with decom and they're working on fixing it, or maybe they realized that now is not a good time, but they want to finish doing decom later.  In those scenarios, they probably don't want to trigger a movement of partitions back to the node.

But I'm not sure we can have a behavior that respects that, without having some kind of indicator for whether a node is "pending removal", which we don't have today :thinking:

Reply:
i think you are right, gonna drop it for now.

Reply:
i dropped this part, if somebody wants rebalancing after decommissioning has been aborted it is always possible to use on demand rebalance